GW2EIEvtcParser/EIData/Mechanics/MechanicData.cs

```diff
@@ -46,10 +46,9 @@ internal MechanicData(List<Mechanic> fightMechanics)
 
         private void ComputeMechanics(ParsedEvtcLog log)
         {
-            var regroupedMobs = new Dictionary<int, AbstractSingleActor>();
             foreach (Mechanic mech in _mechanicLogs.Keys)
             {
-                mech.CheckMechanic(log, _mechanicLogs, regroupedMobs);
+                mech.CheckMechanic(log, _mechanicLogs, new Dictionary<int, AbstractSingleActor>());
```

Comment:
The dictionnary is shared between all mechanics, creating a new one per iteration is not useful.